### PR TITLE
feat: ability to update order branch

### PR DESCRIPTION
**Pull request by AdrianAndersen**

### CODE REVIEW

**Comment by AdrianAndersen** [comment]
[Mulighet til å endre utdelings-filial for ordre](https://trello.com/c/MHNwoBB1/714-mulighet-til-%C3%A5-endre-utdelings-filial-for-ordre)

File: src/app/order/order-detail/order-detail-card/order-detail-card.component.ts [hunk 1]
```diff
@@ -9,7 +12,42 @@ import { Order } from "@boklisten/bl-model";
...
+		private _orderService: OrderService,
+		private _toasterService: ToasterService,
+		private _authService: AuthService
+	) {}
+
+	ngOnInit() {
+		this.isAdmin = this._authService.isAdmin();
+		this.getAllBranches();
+		this.selectedBranch = String(this.order.branch);
+		console.log(this.selectedBranch);
```
Comment (LarsSelbekk) [inline src/app/order/order-detail/order-detail-card/order-detail-card.component.ts:30]:
```suggestion
```

File: src/app/order/order-detail/order-detail-card/order-detail-card.component.ts [hunk 2]
```diff
@@ -9,7 +12,42 @@ import { Order } from "@boklisten/bl-model";
...
+	}
+
+	private async getAllBranches() {
+		this._branchService.get().then((branches: Branch[]) => {
+			this.branches = branches;
+		});
+	}
+
+	public async updateOrderBranch(newBranchId: string) {
```
Comment (LarsSelbekk) [inline src/app/order/order-detail/order-detail-card/order-detail-card.component.ts:39]: I understand it's only necessary to update the order branch for P2P-matching, but would that not have any unintended consequences when the order disagrees with the customeritem? I trust your answer so you can merge if you want to, I just want to know you've thought about it

**Review by LarsSelbekk — APPROVED** [review]
I'm going to approve since we're a bit short on time, but there's a forgotten console-log and a question I need to know you've thought about.

File: src/app/order/order-detail/order-detail-card/order-detail-card.component.ts (hunk 2 above)
Comment (AdrianAndersen) [reply src/app/order/order-detail/order-detail-card/order-detail-card.component.ts:43]: That is a very good point! I updated it so it updates the customerItems as well. (that handoutById (branch) on the customerItem is used for reminders)

**Pull request merged**